person.rb

```diff
@@ -6,13 +6,14 @@ def correct_name
 
 class Person < Nameable
   attr_reader :id
-  attr_accessor :name, :age
+  attr_accessor :name, :age, :rentals
 
   def initialize(age, name = 'Unknown', parent_permission: true)
     @id = Random.rand(1..100)
     @name = name
     @age = age
     @parent_permission = parent_permission
+    @rentals = []
     super()
   end
 
```

Comment:
- [ ] A person has many rentals (the has-many side of Person and Rental) and that means you should add a method for adding rentals to the Person class. Check out the example in this lesson: [link](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/articles/oop_relationships_by_examples.md#many-to-many-relationship
) for more information.